FusionIIIT/applications/scholarships/models.py

```diff
@@ -1,3 +1,269 @@
 # from django.db import models
 
+from datetime import datetime
+
 # Create your models here.
+from django.db import models
+
+from applications.academic_information.models import Student
+from applications.globals.models import ExtraInfo
+
+# Create your models here
+
+
+class Constants:
+    MOTHER_OCC_CHOICES = (
+          ('EMPLOYED', 'EMPLOYED'),
+          ('HOUSE_WIFE', 'HOUSE_WIFE')
+
+    )
+    HOUSE_TYPE_CHOICES = (
+         ('RENTED', 'RENTED'),
+         ('OWNED', 'OWNED')
+
+    )
+    STATUS_CHOICES = (
+        ('COMPLETE', 'COMPLETE'),
+        ('INCOMPLETE', 'INCOMPLETE')
+    )
+
+    FATHER_OCC_CHOICES = (
+        ('Service', (
+                        ('government', 'Government'),
+                        ('private', 'Private'),
+                        ('public', 'Public'),
+                        )
+         ),
+        ('Non_Salaried', (
+                         ('business', 'Business'),
+                         ('medical', 'Medical'),
+                         ('consultant', 'Consultant'),
+                        )
+         ),
+        ('pensioners', 'Pensioners')
+
+    )
+    time = (
+        ('0', '12 Midnight'),
+        ('1am', '1'),
+        ('2am', '2'),
+        ('3am', '3'),
+        ('4am', '4'),
+        ('5am', '5'),
+        ('6am', '6'),
+        ('7am', '7'),
+        ('8am', '8'),
+        ('9am', '9'),
+        ('10am', '10'),
+        ('11am', '11'),
+        ('12 Noon', '12'),
+        ('1pm', '13'),
+        ('2pm', '14'),
+        ('3pm', '15'),
+        ('4pm', '16'),
+        ('5pm', '17'),
+        ('6pm', '18'),
+        ('7pm', '19'),
+        ('8pm', '20'),
+        ('9pm', '21'),
+        ('10pm', '22'),
+        ('11pm', '23'),
+        ('12 Midnight', '0')
+    )
+    batch = (
+        ('UG1', 'UG1'),
+        ('UG2', 'UG2'),
+        ('UG3', 'UG3'),
+        ('UG4', 'UG4'),
+        ('PG1', 'PG1'),
+        ('PG2', 'PG2')
+    )
+
+
+class Mcm(models.Model):
+    email_id = models.CharField(max_length=25)
```

Comment:
Student id has to remain!

Reply:
We are traveling right now in train neccesary changes will be done tomorrow